database_cleanup/README.rst

```diff
@@ -1,15 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================
+Database cleanup
+================
+
 Clean your OpenERP database from remnants of modules, models, columns and
 tables left by uninstalled modules (prior to 7.0) or a homebrew database
 upgrade to a new major version of OpenERP.
 
+Caution! This module is potentially harmful and can *easily* destroy the
+integrity of your data. Do not use if you are not entirely comfortable
+with the technical details of the OpenERP data model of *all* the modules
+that have ever been installed on your database, and do not purge any module,
+model, column or table if you do not know exactly what you are doing.
+
+Usage
+=====
+
 After installation of this module, go to the Settings menu -> Technical ->
 Database cleanup. Go through the modules, models, columns and tables
 entries under this menu (in that order) and find out if there is orphaned data
 in your database. You can either delete entries by line, or sweep all entries
 in one big step (if you are *really* confident).
 
-Caution! This module is potentially harmful and can *easily* destroy the
-integrity of your data. Do not use if you are not entirely comfortable
-with the technical details of the OpenERP data model of *all* the modules
-that have ever been installed on your database, and do not purge any module,
-model, column or table if you do not know exactly what you are doing.
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot/149/9.0
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+* ...
```

Comment:
`Unexpected possible title overline or transition.|Treating it as ordinary text because it's so short.`
Could we delete this unused section?